config/configtelemetry/doc.go

```diff
@@ -4,4 +4,46 @@
 // Package configtelemetry defines various telemetry level for configuration.
 // It enables every component to have access to telemetry level
 // to enable metrics only when necessary.
+//
+// 1. configtelemetry.None
+//
+// No telemetry data should be collected.
+//
+// 2. configtelemetry.Basic
+//
+// Signals associated with this level cover the essential coverage of the component telemetry.
+//
+// This is the default level recommended when running the Collector.
+//
+// The signals associated with this level must show low cardinality, the number of combinations of dimension values. The maximum cardinality allowed is set to 50.
+//
+// The signals associated with this level must represent a small data volume:
+//   - No more than 5 metrics reported.
+//   - At most 1 span actively recording simultaneously, covering the critical path.
+//   - At most 5 log records produced every 30s.
```

Comment:
I question whether logs should be included in this document at all. If they are, I would think guidance is given in terms of severity level, rather than telemetry level.

Log severity and sampling are well established ways to manage log volume and we already have configuration for them in the collector. I think that trying to bring in the concept of telemetry level in any way is going to be confusing at best.

Reply:
I will remove them